assignments/prototypes.js

```diff
@@ -15,13 +15,33 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject(atti){
+  Humanoid.call(this, atti);
+}
+
+CharacterStats.prototype = Object.create(GameObject.prototype);
+
+GameObject.prototype.destroy = function() {
+  return (`${this.name} was removed from the game.`)
+}
+
 
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(attrib){
+  Humanoid.call(this, attrib);
+}
+
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+
+CharacterStats.prototype.takeDamage = function() {
+  return (`${this.name} took damage.`)
```

Comment:
Now that we are trying to make the child we need to use the 
```js
CharacterStats.prototype = Object.create(GameObject.prototype);
```

Which gives the child all the parents functions in the prototype, and then you can add your own new ones just for the child like you do on line 41.